bin/scalafmt

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
```

Comment:
CI vs scalafmt is really complicated story. The default scalafmt command doesn't work for the main repo at all, so we have to use non-default entry point that works a better.